### ISSUE TITLE
When user is in full screen view of video and he clicks back button it comes out of the APP, but it should come out of the full screen mode. This makes user login again and again

### ISSUE THREAD

**Issue by anand0205** [WWBN/AVideo-Encoder#269]

**Comment by anand0205** [comment]
[Record_2020-05-27-07-38-48_24e593c7bba62d63c6beb65449d4de2b.zip](https://github.com/WWBN/AVideo-Encoder/files/4686086/Record_2020-05-27-07-38-48_24e593c7bba62d63c6beb65449d4de2b.zip)

**Comment by DanielnetoDotCom** [comment]
I see,

I tried on my device and it works fine. but I notice the same behavior if I press the back button twice. but quit the app is an expected behavior


Are you sure you did not press it twice? Did you try on another device? I need to know if it is related to the device or Android version.

**Comment by anand0205** [comment]
No Daniel for sure after pressing back only once it comesa out..Try with a video upload as MP4..we have tested it on multiple android devices, similar behaviour is observed

**Comment by DanielnetoDotCom** [comment]
I am trying to find a solution but not much progress.
on my devices some times this happens some times it does not.

BTW, we have a button to quit the fullscreen on the bottom

**Comment by anand0205** [comment]
Yes we do have a button for that, but i can't ask customers not to click back from device, Please check if we can do something with this issue. As this is not given everyone a pleasant experience of app. Thank you Daniel !
